### ISSUE TITLE
6 signs of a Rule Breaker

### ISSUE THREAD

**Issue by sp500** [sp500/investment#3]
6 signs of a Rule Breaker
Ever wonder how you can invest in the next big thing — before everyone knows it's the next big thing?
That's Motley Fool co-founder David Gardner's specialty, and he's built an astounding track record by investing in up-and-coming companies that often go on to disrupt entire industries. (David's Motley Fool Rule Breakers service is beating the market by 84 percentage points since 2004.)
So what's David Gardner's secret?
The idea is simple: Invest in great companies early in their high-growth stages and then hold them for the long term. This will provide you the highest possible returns. Period.
We call these great companies Rule Breakers, and David has identified six traits to help you find tomorrow's disruptors today.
Not every great growth investment will have all of these traits, but the companies that do deserve your special attention. They're the businesses that are most likely to sustain extraordinary growth over a long period of time.
1. Top dog and first mover in an important, emerging industry
A top dog holds the dominant market share in its industry; usually it's the largest by market capitalization. The first mover is the innovator that first exploits a niche — essentially creating its market. And finally, that niche must actually be worth dominating.
Who's put all of this together? Take Starbucks. Starbucks didn't invent the coffee shop, but it was the first to conceive of its businesses on a national and ultimately international scale — even as others didn't see growth opportunities there.
Rule Breakers aren't hidden; they're right there before our eyes, bringing disruptive technology, clever and effective marketing, or a brand-new business model.
2. Sustainable advantage gained through business momentum, patent protection, visionary leadership, or inept competitors
Successful businesses attract competition. The critical question is how well a company can fend off that competition.
In some businesses, like the pharmaceutical industry, patents can enforce a lasting competitive advantage. On the other hand, patent protection can be problematic in the software industry, where protected inventions can often be worked around.
Luckily, there are other ways of protecting a competitive advantage. Companies have trade secrets (the formula for Coke isn't patented; it's a well-guarded secret known to only a few employees), and they can build expertise that others find hard to duplicate. Some businesses require daunting levels of capital investment to establish, while others invest in their reputations and brand names. Sometimes a company's leaders are just smarter than the competition — and sometimes competitors find they just can't adapt to a changing world.
The key is to find what we call a company's moat — its bulwark against inevitable competitors — and figure out how many alligators are in it.
3. Strong past price appreciation
Consider an investor's take on Newton's law of inertia: A stock on the rise tends to remain on the rise unless an outside force disrupts its path.
The best growth stocks continue rising, because their advantages allow them to sustain remarkable earnings and cash flow growth and to continuously win new converts among the ranks of both customers and investors. Don't count on momentum to save your bacon in the absence of other strong fundamentals. But a strong company firing on all cylinders can sustain a remarkably extended run.
4. Good management and smart backing
Good management trumps almost all other concerns. Thanks to good management, a company with few structural advantages over its rivals can end up being very successful and returning a lot of value to shareholders. (Better a mediocre business with great management than a great business with mediocre management. Over time, those latter guys will screw up a free lunch.)
Now imagine adding great management to a great company — it's a powerful force.
Judging the quality of a management team is a bit subjective, but that's because it's human beings who head these companies. Luckily, we're human beings, too, and most of us are equipped with skills to assess the more subjective aspects. Listen to conference calls and investor presentations. Even if you can't talk to management directly, the Internet makes it easy to hear how the top brass thinks and how they interact with investors. Are they smart? Visionary? Inspiring? The heads of Rule Breaking companies are often career entrepreneurs with a track record of business formation you can look to. Even if you can't put a number on it, you can certainly get some idea of whom you're dealing with.
5. Strong consumer appeal
It's almost impossible to overstate the power of a strong brand. If a business has mass consumer appeal, sustaining extraordinary growth is that much easier. A brand eventually reinforces itself — that's why a company like Starbucks doesn't really need much advertising. A brand also becomes associated with an experience. We're creatures of habit, and when we have to think less, it makes our lives seem easier. The habit that comes from a strong brand — knowing where your next cup of coffee is coming from — immeasurably strengthens a company against its competitors. It also gives a company pricing power over rivals — you expect to pay more for a brand name, right?
Of course, some great companies work in specialty businesses that simply don't have mass consumer appeal. That's OK, but we want to know that the company's product, name, and reputation constitute a brand among the people who matter. If you're looking at an esoteric software business, ask yourself this question: Could this company price its product 5% or 10% higher than its competitors and still maintain market share because of its reputation and loyal customers?
6. Grossly overvalued according to the financial media
This might sound like an odd factor. Who wants to buy a stock that those wise financial commentators say is too expensive and poised for a tumble?
In fact, being derided as overvalued is a trait shared by many Rule Breakers that supposedly smart investors avoid ... stocks that go on to double, triple, quintuple, and more over the years. The "too expensive" label comes from underestimating how a Rule Breaker can disrupt its industry, displace competitors, and grow over a relatively short time. Investors' fears leave many on the sidelines, only to come in later and drive the stock up further as the writing on the wall becomes more apparent.
The Foolish Bottom Line
Where are the creative disruptions happening today? Some are undoubtedly just bubbling up now, invisible to all but a few eyes. But others have matured enough to become apparent to investors willing to break a few rules. Are those opportunities in biotechnology? In robotics? In cloud computing? Maybe all of the above?
Finding these companies will challenge your sleuthing abilities. Buying them will challenge to your well-honed impulses as a cautious investor.
Just remember this: Investing in Rule Breakers isn't about taking giant gambles on unproven, blue-sky ideas. It's about recognizing the companies that are already succeeding in creating a new niche — and identifying the ones that are going to keep succeeding tomorrow.
In other words, it's the ultimate in growth investing.
